voskcli/transcribe.py

```diff
@@ -67,6 +75,21 @@ def model_path(model):
 
 
 def write_captions_paragraph(vtt, paragraph):
+    '''
+    Append time-coded paragraphs to vtt file.
+    
+    :param vtt: Empty or partially filled WebVTT file to write transcriptions to
+    :type vtt: WebVTT
+    :param paragraph: Transcription data segment containing 1-2 lines of up to 35 chars.
```

Comment:
not sure if detailled description of the paragraph data should be ommited

Reply:
> not sure if detailled description of the paragraph data (see lines below) should be ommited

I like it. It would help me, if I would want to use this method. It makes it easier to figure out what structure I need to pass to the parameter.

> containing 1-2 lines of up to 35 chars

That's not something this function is limited to, is it? This function has no limit to how long captions can be. The limit is enforced somewhere else in the tool but is not related to this. I would stick to what this function does.